.github/workflows/test-languages.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Test Languages
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Test Languages
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout PLCC
+        uses: actions/checkout@v3
+        with:
+          repository: ourPLCC/plcc
+      -
+        name: Build test image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./containers/plcc/Dockerfile
+          tags: test-image:latest
+          push: false
+      -
+        name: Test Languages
+        uses: addnab/docker-run-action@v3
+        with:
+          image: test-image:latest
+          run: |
+            git clone https://github.com/ourPLCC/languages.git
+            ./languages/bin/test.bash
+            rm -rf languages
```

Comment:
No need to clean up. All of this is thrown away after the test completes (success or failure).